robolectric/src/main/java/org/robolectric/RobolectricTestRunner.java

```diff
@@ -38,23 +34,13 @@
 import org.robolectric.util.Pair;
 import org.robolectric.util.ReflectionHelpers;
 
-import java.io.File;
-import java.io.IOException;
-import java.io.InputStream;
-import java.lang.annotation.Annotation;
+import java.io.*;
```

Comment:
What's our feeling on imports? We'll end up with lots of churn if we don't standardize?